### PR TITLE
VIH-9550 Retry booking should attempt to rebook the conference in video api

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9950

### Change description ###
Changes the functionality of the retry button for a failed booking from confirmHearing to rebookHearing. This will attempt to re-create the conference in video API (along with any users that failed to create) and then poll the booking status to verify that it was created.

The polling is the same as what we use on the summary component to check whether the booking was initially created successfully, so it has been refactored into a common booking status service.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/vh-admin-web/values.dev.template.yaml [hunk 1]
```diff
@@ -8,4 +8,5 @@ java:
     AZUREAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
     AZUREAD__REDIRECTURI: https://${SERVICE_FQDN}/home
     DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+    VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api-pr-663.dev.platform.hmcts.net/
```
Comment (oliver-scott) [inline charts/vh-admin-web/values.dev.template.yaml:12]: note to self - remove before merge

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts [hunk 2]
```diff
@@ -253,13 +255,8 @@ export class SummaryComponent implements OnInit, OnDestroy {
...
 
                 if (this.judgeAssigned) {
-                    // Poll Video-Api for booking confirmation
-                    const schedule = timer(0, 5000).subscribe(async counter => {
-                        const hearingStatusResponse = await this.hearingService.getStatus(hearingDetailsResponse.id);
-                        if (hearingStatusResponse?.success || counter === 10) {
-                            schedule.unsubscribe();
-                            await this.processBooking(hearingDetailsResponse, hearingStatusResponse);
-                        }
+                    this.bookingStatusService.pollForStatus(hearingDetailsResponse.id).subscribe(async response => {
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts:258]: Nice!

**Review by shaed-parkar — APPROVED** [review]
Nice refractor on the polling!

**Review by phildebrah — APPROVED** [review]
Test and working successfully

**Pull request merged**